operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -989,134 +989,354 @@ compactor:
   env: []
   extraEnvFrom: []
 
-memcached:
+memcached-chunks:
+  # -- Specifies whether memcached should be enabled
   enabled: false
-  architecture: high-availability
-  arguments:
-    - -m 8192
-    - -o
-    - modern
-    - -v
-    - -I 1m
-    - -c 4096
+
+  # -- Total number of memcached replicas
+  replicas: 1
+
+  # -- Container image settings for memcached
   image:
     repository: memcached
-    tag: 1.6.9
-  # maxItemMemory is in bytes. Should match memcached -I flag (which is in MB)
-  # It is a string to avoid https://github.com/helm/helm/issues/1707.
-  maxItemMemory: '1048576'  # (* 1 (* 1024 1024))
-  metrics:
-    enabled: true
-    image:
-      registry: quay.io
-      repository: prometheus/memcached-exporter
-      tag: v0.9.0
-  replicaCount: 1
+    tag: 1.6.9-alpine
```

Comment:
Loki has a common `memcached` section for setting the image in one place and a couple of other things. Wondering if we should do the same.

Reply:
following loki inspiration, new [memcached](https://github.com/grafana/mimir/blob/88fb1e3e1c041ea5e639ce75f8d0cb4bd461ddd9/operations/helm/charts/mimir-distributed/values.yaml#L1009-L1031) and [memcachedExporter](https://github.com/grafana/mimir/blob/88fb1e3e1c041ea5e639ce75f8d0cb4bd461ddd9/operations/helm/charts/mimir-distributed/values.yaml#L1033-L1040) sections were added to share common configuration across memcached clusters